bindings/java/src/org/sleuthkit/datamodel/HostAddress.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Sleuth Kit Data Model
+ *
+ * Copyright 2020 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.datamodel;
+
+import java.util.Objects;
+
+/**
+ *
+ * Abstracts an address associated with a host. A host may have multiple
+ * addressed of different types associated with it ant a give time.
+ */
+public class HostAddress /* RAMAN TBD: implements Content */ {
```

Comment:
This file is not part of OS Accounts.  It got in here unintentionally, since I'd already created it on this branch.  You may ignore it for the purpose of reviewing this PR.

Reply:
Can you just remove the file from this branch?